ova/build.sh

```diff
@@ -23,6 +23,9 @@ ROOT_WORK_DIR="/go/src/github.com/vmware/dispatch/"
 ROOT_INSTALLER_DIR="${ROOT_DIR}/ova"
 ROOT_INSTALLER_WORK_DIR="${ROOT_WORK_DIR}/ova"
 
+ROOT_UI_DIR="$GOPATH/src/github.com/vmware/dispatch-ui-binaries"
+ROOT_WORK_UI_DIR="/go/src/github.com/vmware/dispatch-ui-binaries"
```

Comment:
how about just `ui` directory in dispatch directory?
`ROOT_WORK_UI_DIR="${ROOT_DIR}/ova`
`ROOT_DIR` is already mounted in local docker, and you keep everything inside dispatch.
Using `/go/src/github.com/vmware/dispatch-ui-binaries` kind of suggests it's a go-dependent directory.